### PR TITLE
Cleans up the repository, adding molecule tests.

### LINKED ISSUE #5: Using outdated gpg key
THe preferred gpg key for Passenger builds is now https://packagecloud.io/phusion/passenger/gpgkey

### PR DESCRIPTION
**Pull request by bibliotechy**
Allows specific version of passenger to be provided.

Updates the gpgkey passenger uses for signing repository.

Fixes #5 and #6

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = "*"
+
```
Comment (cmharlow) [inline Pipfile]: i believe you need to add `docker = "*"` here for tests to run in shell / isolated env (required for `molecule destroy`

Comment (bibliotechy) [reply Pipfile]: @cmharlow  So, I just tried this locally, and `molecule test` fails at the destroy step when I have `docker = "*"` in `[dev-packages]` and not in `[`packages]`.

Comment (cmharlow) [reply Pipfile]: these are dev env requirements, not prod requirements, so lets figure it out.

Comment (cmharlow) [reply Pipfile]: putting a PR against this to raise the discussion on handling molecule / ansible dev requirements vs prod (runtime?) requirements.

Comment (bibliotechy) [reply Pipfile]: No, I was running without the `--dev` flag so that was my issue. Agree that there are no runtime requirements for docker.

**Pull request merged**